src/components/NcEmojiPicker/NcEmojiPicker.vue

```diff
@@ -187,17 +201,24 @@ This component allows the user to pick an emoji.
 </template>
 
 <script>
-import NcPopover from '../NcPopover/index.js'
-import NcTextField from '../NcTextField/index.js'
+import { getBuilder } from '@nextcloud/browser-storage'
+import { Picker, Emoji, EmojiIndex } from 'emoji-mart-vue-fast'
 import { t } from '../../l10n.js'
+import { Color } from '../../utils/GenColors.js'
 
-import { Picker, Emoji, EmojiIndex } from 'emoji-mart-vue-fast'
 import data from 'emoji-mart-vue-fast/data/all.json'
+import IconCircle from 'vue-material-design-icons/Circle.vue'
+import NcButton from '../NcButton/index.js'
+import NcColorPicker from '../NcColorPicker/NcColorPicker.vue'
+import NcPopover from '../NcPopover/index.js'
+import NcTextField from '../NcTextField/index.js'
 
-// Shared emoji index for all NcEmojiPicker instances
+// Shared emoji index and skinTone for all NcEmojiPicker instances
 // Will be initialized on the first NcEmojiPicker creating
 let emojiIndex
 
+const storage = getBuilder(`${appName}`).persist(true).build()
```

Comment:
I'm not sure it should be per-app storage. If a user has chosen the color in one app on one page, they would expect it to work on other pages.

Reply:
Yep, fully agree, should be saved per-account. :)

Reply:
Then use a generic scope. :white_check_mark: